ttnn/cpp/ttnn/operations/pool/downsample/downsample_pybind.hpp

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
```

Comment:
see ccl/all_gather. it uses pybind_fwd to avoid using heavy includes. please follow the same pattern

Reply:
Sorry missed in rebase. Done now.